sql_scripts/filter_answer_view.sql

```diff
@@ -1,13 +1,13 @@
-select distinct 
+select distinct
   a_t.user_id,
   a_t.display_name,
   a_t.answer_id,
   a_t.question_id,
   a_t.is_accepted,
   a_t.score,
   a_t.answer_date,
-  a_t.link,
```

Comment:
Removal of `a_t.link` is not expected. Check then run tests to make sure this appears.